tests/integration/conftest.py

```diff
@@ -33,6 +36,23 @@ def use_temp_profile():
         switch_default_profile(current_profile_name)
 
 
+@pytest.fixture(scope="session")
+def integration_test_profile():
```

Comment:
What is this for? It looks like we are making two integration test profiles now?

Reply:
command_runner shouldn't be needed anymore with this fixture in place. With `command_runner` we were setting up and tearing down the test profile every single test, this just does it once for the session. This also sets the password, which means we don't need to use pyexpect to handle the prompt.

Reply:
This is same as the other method you created, the only difference is now its a fixture, I am planning to remove one of them, but there were subtle changes in the approach so I created a new method so that it doesn't affect existing tests. The scope here was suppose to be `send-to` only.

Reply:
The runner seems the way to go, hopefully it is pretty simple to get rid of the other method and swap it all out

Reply:
Done.